thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java

```diff
@@ -539,6 +549,22 @@ private String getDimensions(String dataset, String exploreDimensions) throws Ex
     return reorderedExploreDimensions.toString();
   }
 
+  // Activate anomaly function
+  @POST
+  @Path("/anomaly-function/activate")
+  public Response activateAnomalyFunction(@NotNull @QueryParam("functionId") Long id, @QueryParam("modifiedFunctionName") String functionName){
```

Comment:
I believe if you input a functionName that already used, you will not be successful in updating the AnomalyFunctionDAO. Why you need to rename the function anyway if you claim this endpoint is to activate a function (you just need the function id or function name, both are unique identifier). It is hard to believe that this simple but useful endpoint is not yet available though; thanks for implementing this.